ts/Series/Organization/OrganizationSeries.ts

```diff
@@ -509,34 +570,47 @@ class OrganizationSeries extends SankeySeries {
             levelOptions: OrganizationSeriesLevelOptions =
                 (series.mapOptionsToLevel as any)[level || 0] || {},
             options = point.options,
-            stateOptions: OrganizationSeriesOptions = (
-                levelOptions.states &&
-                (levelOptions.states as any)[state as any]
-            ) || {},
-            values: (
-                OrganizationPointOptions &
-                OrganizationSeriesOptions
-            ) = ['borderRadius', 'linkColor', 'linkLineWidth']
-                .reduce(function (
-                    obj: Record<string, unknown>,
-                    key: string
-                ): Record<string, unknown> {
-                    obj[key] = pick(
-                        (stateOptions as any)[key],
-                        (options as any)[key],
-                        (levelOptions as any)[key],
-                        (series.options as any)[key]
-                    );
-                    return obj;
-                }, {});
+            stateOptions: OrganizationSeriesOptions =
+                (levelOptions.states &&
+                    (levelOptions.states as any)[state as any]) ||
+                {},
+            borderRadius = pick(
+                stateOptions.borderRadius,
+                options.borderRadius,
+                levelOptions.borderRadius,
+                series.options.borderRadius
+            ),
+
+            linkColor = pick(
```

Comment:
¯\_(ツ)_/¯. @pawelfus  what do you think?

Reply:
I don't know if it really is the best way. @pawelfus  what is your take on this?

Reply:
No strong opinion here yet, with `reduce` we have castings to `any`. Without, we have more code. I'm leaning towards new solution, because `borderRadius` has fewer options, so `reduce` would end up with just two options.